### PR TITLE
Add basic typescript support

### PR DESCRIPTION
**Pull request by eloiqs**
Hey there!

Typescript is awesome for third party libraries. Lets encourage its use!

This is a basic prototype, I didn't put much thought into it and just wanted to get something basic going so others can improve on top.

Feel free to use it as you want! 😄 

example usage:
```
javali my-ts-lib --ts
```

### CODE REVIEW

**Comment by diogomoretti** [comment]
Thanks for your PR @eloiqs 😉 Could you put the instructions in readme file?

**Comment by eloiqs** [comment]
I added a mention and example in the readme!

I also improved the implementation a bit. Let me know what you think! 😄

File: cli/index.js [hunk 1]
```diff
@@ -5,13 +5,20 @@ const create = require('./scripts/create')
 const log = require('./scripts/log')
 const { version } = require('../package.json')
 
+let appName
+
 program
   .version(version)
+  .arguments('<appName>')
   .option('-v, --version', 'version')
+  .option('--typescript', 'use typescript')
```
Comment (diogomoretti) [inline cli/index.js]: Add `-t` short param

**Review by diogomoretti — CHANGES_REQUESTED** [review]

**Review by diogomoretti — APPROVED** [review]

**Comment by diogomoretti** [comment]
[Javali@1.1.0 was released](https://www.npmjs.com/package/javali/v/1.1.0). Thanks!

**Pull request merged**